geps/gep-1731/index.md

```diff
@@ -0,0 +1,436 @@
+# GEP-1731: Configurable Retries
+
+* Issue: [#1731](https://github.com/kubernetes-sigs/gateway-api/issues/1731)
+* Status: Implementable
+
+(See status definitions [here](/geps/overview/#gep-states).)
+
+## TLDR
+
+To allow configuration of a Gateway to retry unsuccessful requests to backends before sending a response to a client request.
+
+## Goals
+
+* To allow specification of [HTTP status codes](https://www.rfc-editor.org/rfc/rfc9110#name-overview-of-status-codes) for which a request should be retried.
+* To allow specification of the maximum number of times to retry a request.
+* To allow specification of the minimum backoff interval between retry attempts.
+* To define any interaction with configured HTTPRoute [timeouts](/geps/gep-1742/).
+* Retry configuration must be applicable to most known Gateway API implementations.
+
+## Future Goals
+
+* To allow specification of gRPC status codes for which a request should be retried.
```

Comment:
Although it's annoying, I think it may be easier to keep GRPCRoute retries in a separate GEP so that doesn't slow down the graduation of these fields.